R/metrics.R

```diff
@@ -1,4 +1,65 @@
 utils::globalVariables(c("PredictionSurv"))
+#' Calculate integrated metrics based on time-depenent metrics.
+#'
+#' This function allows for calculation of integrated metrics based on a time dependent metric. A possibility to cut off the data at certain quantiles is implemented, as well as weighting the integrated metric by max time
+#'
+#' @param loss_function - A time dependent loss function taking arguments (y_true, risk, surv, times)
+#'
+#' @param ... - other parameters, currently ignored
+#' @param normalization - either NULL or "t_max". Decides what kind of weighting should be applied to the integrated metric. If "t_max" then the integral is calculated using dw(t), where w(t) = t/t_max. If NULL (default) the integral is calculated using dt.
```

Comment:
Parameter `normalization` can be also set to `"survival"` - it has to be added do documentation. It's also a good idea to add references to these implemented integration methods.

Reply:
I've corrected the description by myself